core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -185,18 +216,60 @@ public PlayerInformation getPlayerInformation() {
     public void attach(PlayerView playerView) {
         videoContainer = VideoContainer.with(playerView.getContainerView());
         playerView.simplePlayerView().setPlayer(exoPlayer);
-        // TODO : Set SubtitleView on the facade.
     }
 
     @Override
-    public void selectAudioTrack(int audioTrackIndex) {
-        //TODO: Select the audio track on the facade.
+    public void selectAudioTrack(PlayerAudioTrack audioTrack) {
+        MappingTrackSelector.MappedTrackInfo trackInfo = trackSelector.getCurrentMappedTrackInfo();
+        TrackGroupArray trackGroups = trackInfo.getTrackGroups(TRACK_TYPE_AUDIO);
+        FixedTrackSelection.Factory factory = new FixedTrackSelection.Factory();
+
+        MappingTrackSelector.SelectionOverride selectionOverride = new MappingTrackSelector.SelectionOverride(
+                factory,
+                audioTrack.groupIndex(),
+                audioTrack.trackIndex()
+        );
+        trackSelector.setSelectionOverride(TRACK_TYPE_AUDIO, trackGroups, selectionOverride);
```

Comment:
Can we simplify all this code and just do `trackSelector.selectAudioTrack(audioTrack)`?  if trackSelector is a native library I would suggest to wrap it into our own class so that we can test it in isolation from this class